esmvaltool/diag_scripts/thermodyn_diagtool/lorenz_cycle.py

```diff
@@ -505,51 +517,54 @@ def mka2k(wap, t_t, w_g, t_g, p_l):
     """Compute the KE to APE energy conversions from t and w.
 
     Arguments:
-    - wap: a 3D vertical velocity field;
-    - t_t: a 3D temperature field;
-    - w_g: a vertical velocity vertical profile;
-    - t_g: a temperature vertical profile;
-    - p_l: the pressure levels;
+    ----------
+    wap: a 3D vertical velocity field;
+    t_t: a 3D temperature field;
+    w_g: a vertical velocity vertical profile;
+    t_g: a temperature vertical profile;
+    p_l: the pressure levels;
     """
-    a2k = -(R / p_l[:, np.newaxis, np.newaxis] *
-            (t_t * np.conj(wap) + np.conj(t_t) * wap))
-    a2k[:, :, 0] = -(R / p_l[:, np.newaxis] *
-                     (t_t[:, :, 0] - t_g[:, np.newaxis]) *
-                     (wap[:, :, 0] - w_g[:, np.newaxis]))
+    a2k = - np.real(R / p_l[:, np.newaxis, np.newaxis] *
+                    (t_t * np.conj(wap) + np.conj(t_t) * wap))
+    a2k[:, :, 0] = - np.real(R / p_l[:, np.newaxis] *
+                             (t_t[:, :, 0] - t_g[:, np.newaxis]) *
+                             (wap[:, :, 0] - w_g[:, np.newaxis]))
     return a2k
 
 
 def mkaeaz(v_t, wap, t_t, ttt, ttg, p_l, lat, gam, nlat, nlev):
     """Compute the zonal mean - eddy APE conversions from t and v.
 
     Arguments:
-    - v_t: a 3D meridional velocity field;
-    - wap: a 3D vertical velocity field;
-    - t_t: a 3D temperature field;
-    - ttt: a climatological mean 3D temperature field;
-    - p_l: the pressure levels;
-    - lat: the latudinal dimension;
-    - gam: a vertical profile of the stability parameter;
-    - nlat: the number of latitudes;
-    - nlev: the number of levels;
+    ----------
+    v_t: a 3D meridional velocity field;
+    wap: a 3D vertical velocity field;
+    t_t: a 3D temperature field;
+    ttt: a climatological mean 3D temperature field;
+    p_l: the pressure levels;
+    lat: the latudinal dimension;
+    gam: a vertical profile of the stability parameter;
+    nlat: the number of latitudes;
+    nlev: the number of levels;
     """
     dtdp = np.zeros([nlev, nlat])
     dtdy = np.zeros([nlev, nlat])
+    ttt = np.real(ttt)
     for l_l in np.arange(nlev):
         if l_l == 0:
-            t_1 = np.real(ttt[l_l, :, 0]) - ttg[l_l]
-            t_2 = np.real(ttt[l_l + 1, :, 0]) - ttg[l_l + 1]
+            t_1 = ttt[l_l, :, 0] - ttg[l_l]
```

Comment:
In general try to work with more descriptive and human readable names for you variables. Right now it is very hard to follow what happens in these calculations. The codacy issue that tells you to use three or more characters per variable name is meant for that: not to force you to use three characters but to make the code readable.